src/Routes/Tasks.js

```diff
@@ -0,0 +1,54 @@
+const BaseRoute = require('../Structure/BaseRoute');
+const handleError = require('../Util/handleError');
+
+class TasksRoute extends BaseRoute {
+    constructor(client, db, jobs) {
+        super('/tasks');
+        this.router = require('express').Router();
+        this.client = client;
+        this.db = db;
+        this.jobs = jobs;
+        this.routes();
+    }
+
+    routes() {
+        this.router.use(this.requiresAuth.bind(this), this.isMod.bind(this));
+
+        this.router.get('/', (req, res) => {
+            const tasks = this.jobs.map((job, idx) => {
+              return {
+                id: idx,
+                name: job.name,
+                nextInvocation: job.nextInvocation()
+              }
+            });
+            res.render('tasks', { tasks });
+        });
+
+        this.router.get('/run/:id', this.requiresAuth.bind(this), this.isAdmin.bind(this), (req, res) => {
```

Comment:
This isn't really getting anything, would it not be more conventional to make this a POST route?

Reply:
I don't understand why POST would make it more conventional?
POST is normally used to send data with the request.

Reply:
As this is triggering an action, a GET request could end up being cached and not actually triggering the action in the server, whereas a POST will always work.